ipaserver/plugins/baseldap.py

```diff
@@ -550,6 +550,7 @@ class LDAPObject(Object):
     uuid_attribute = ''
     attribute_members = {}
     rdn_is_primary_key = False # Do we need RDN change to do a rename?
+    allow_rename = False
```

Comment:
IMO the attribute is entirely ill, as it does 2 different things: it allows renaming *and* forces DN update on rename.

I think we should finally get rid of it and use
* the `allow_rename` attribute to allow renaming and
* decide whether a DN update is required or not based on the current DN.